### PR TITLE
Add distinctive colors to sell tab

### PR DESCRIPTION
**Pull request by CosmicCoder96**
##### Description
Attempts to add a distinctive negative color to Sell tab on `/exchange`

##### Checklist
- [x] Linter status: 100% pass
- [x] Changes don't break existing behavior
- [x] Tests coverage hasn't decreased

##### Affected core subsystem(s)
UI

##### Screenshots (Optional)
image

##### Refers/Fixes
Fixes: #313

### CODE REVIEW

**Comment by CosmicCoder96** [comment]
@nitinrgupta @pelsasser I am stuck on a issue, the tabs are generated using the external dependency [(ant design)](https://ant.design/docs/react/introduce) . It generates the tab internally and uses the `@primary-color` while rendering. I managed to change the color of the button, using styling. How should I go about the tab border color?

**Comment by nitinrgupta** [comment]
@CosmicCoder96 Thanks for the WIP PR. 
You can check out the properties of `tab` component -> https://ant.design/components/tabs/
See if you can leverage any tab properties, for eg: `tabBarStyle`

**Comment by CosmicCoder96** [comment]
@nitinrgupta `tabBarStyle` again controls the `Tabs` component as a whole, not the individual `TabPane`(s), I tried it earlier. I am currently working on an approach where an action gets triggered when `sell` tab is active and changes the color.

**Comment by nitinrgupta** [comment]
Sounds good. Keep us posted

**Comment by CosmicCoder96** [comment]
@nitinrgupta So this solution works and takes care of the following:
When I am on Buy Tab -> original state of this app is maintained, however while hovering on sell tab, negative state color appears.
When I am on Sell tab -> Negative state is maintained, however when I hover on Buy, primary colors indicating buy are shown.
Behavior GIF:
https://media.giphy.com/media/g0mo31Xbbsu7fAWPpI/giphy.gif

File: src/less/MarketTheme.less [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /* Colors */
 @body-background : #0E121A;
 @primary-color : #00E2C1;
+@negative-state-color: #ff8e1d;
```
Comment (nitinrgupta) [inline src/less/MarketTheme.less]: @CosmicCoder96 Could you please rename `@negative-state-color` to `@secondary-color`? Thanks!

**Comment by CosmicCoder96** [comment]
@nitinrgupta Thanks for the review, Done 👍

**Pull request merged**